.github/workflows/validate.yml

```diff
@@ -16,11 +16,7 @@ jobs:
       NEXT_PUBLIC_GA_ID: mock-ga-id
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-node@v3
       - uses: oven-sh/setup-bun@v1
-        with:
-          node-version: 16
-          registry-url: https://registry.npmjs.org/
-      - run: npm ci --include dev
-      - run: npm run test
-      - run: npm run build
+      - run: bun install
```

Comment:
oops

Reply:
it was on purpose. both your change and this one made sense (although push in opposite directions)

- you removed bun install. this was fine, although it would have been equally ok to remove npm install instead
- mine removes npm install, adds bun install, but goes even further and removes installation of node runtime as well (no need for both!)

Reply:
Oh, I see. Is just that npm install can sometimes update libraries that might break pipelines, where npm ci never tried to update.

Reply:
Good point! Need to add the flag '--frozen-lockfile' to bun install.